Don't install the service files. Auto-detection is used to determine what files
to install, and where, which can be unpredictable on Gentoo. The init.d script
is also not suitable for Gentoo.

diff --git a/setup.py b/setup.py
index 529b591b1..ad23d6ecc 100755
--- a/setup.py
+++ b/setup.py
@@ -187,8 +187,8 @@ rfb_ENABLED = DEFAULT
 quic_ENABLED = DEFAULT
 ssh_ENABLED = DEFAULT
 http_ENABLED = DEFAULT
-service_ENABLED = LINUX and server_ENABLED
-sd_listen_ENABLED = POSIX and pkg_config_ok("--exists", "libsystemd")
+service_ENABLED = False
+sd_listen_ENABLED = False
 proxy_ENABLED  = DEFAULT
 client_ENABLED = DEFAULT
 scripts_ENABLED = not WIN32
